src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -2688,39 +2878,28 @@ void AutomationInstrumentClipView::initParameterSelection() {
 
 	initPadSelection();
 
-	clip->lastSelectedParamID = kNoLastSelectedParamID;
+	clip->lastSelectedParamID = kNoSelection;
 	clip->lastSelectedParamKind = Param::Kind::NONE;
-	clip->lastSelectedParamShortcutX = kNoLastSelectedParamShortcut;
-	clip->lastSelectedParamShortcutY = kNoLastSelectedParamShortcut;
+	clip->lastSelectedParamShortcutX = kNoSelection;
+	clip->lastSelectedParamShortcutY = kNoSelection;
 	clip->lastSelectedParamArrayPosition = 0;
 
+	//if we're going back to the Automation Overview, set the display to show "Automation Overview"
+	//and update the knob indicator levels to match the master FX button selected
 	display->cancelPopup();
-
```

Comment:
this code is no longer needed. it was needed previously to redraw midi channel info, but as it has been replaced with "AUTO" instead, there is no need to redraw channel info. User can go back to regular clip view to see / change midi channel.